test/onnx/model_defs/rnn_model_with_packed_sequence.py

```diff
@@ -15,3 +15,29 @@ def forward(self, input, *args):
         ret, rets = rets[0], rets[1:]
         ret, _ = rnn_utils.pad_packed_sequence(ret, self.batch_first)
         return tuple([ret] + list(rets))
+
```

Comment:
Do we still need to keep RnnModelWithPackedSequence?

Reply:
I see it was used in test_pytorch_onnx_caffe2.py so I'd like to keep it there.